test/parallel/test-http-set-max-idle-http-parser.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const httpCommon = require('_http_common');
+const http = require('http');
+
```

Comment:
Perhaps we should assert that `httpCommon.parsers.max` is not already set to 1, to ensure that the function is doing what it should.

Reply:
Done !